.evergreen/config.yml

```diff
@@ -139,7 +139,7 @@ tasks:
       - func: "run typescript"
         vars:
           TS_VERSION: "4.0.2"
-          TRY_COMPILING_LIBRARY: "true"
+          TRY_COMPILING_LIBRARY: "false"
```

Comment:
Since I'm leaning on the improved `in` narrowing "old TS" [cannot compile the library anymore](https://parsley.mongodb.com/evergreen/node_bson_lint_check_typescript_oldest_patch_5103e4d50cbb27feeb27885cbf44c1c7888389a2_638faabdd6d80a44953c067c_22_12_06_20_49_02/0/task?bookmarks=0,200&selectedLine=178), but the public types continue to work